setup.py

```diff
@@ -43,7 +45,7 @@
     'protobuf>=3.0.0, <4.0dev',
 ]
 
-with open('README.rst', 'r') as readme:
+with io.open('README.rst', 'r') as readme:
```

Comment:
`io.open` treats files as text, and is also equivalent to built-in `open` on Python 3.